pkg/scheduler/framework/interface.go

```diff
@@ -105,6 +106,27 @@ const (
 	MaxTotalScore int64 = math.MaxInt64
 )
 
+// PodsToActivateKey is a reserved state key for stashing pods that will be
+// moved to activeQ if they're present in unschedulableQ or backoffQ.
```

Comment:
Can you update the comment to clarify that adding pods to this state doesn't guarantee that they get activated, and clarify when they will actually get activated.

Reply:
I wonder why use the State instead of having a dedicated method in the framework?

Reply:
The rationale behind this is: the `toActivedPods` are transient and only effective within a scheduling/binding cycle. So manipulating the reserved CycleState make the logic straightforward, and consistent for both in-tree and out-of-tree plugins - they're using the same interface, no hidden API. While using a dedicated method (like `handle.ActivatePods()`), we have to still implement the API as, probably a cycle-specific data struct, and then make the semantics understood by plugin developers, which is not less intuitional IMO.